src/AspNet.Security.OAuth.Deezer/DeezerAuthenticationHandler.cs

```diff
@@ -107,25 +112,24 @@ protected override string BuildChallengeUrl(AuthenticationProperties properties,
 
             var parameters = new Dictionary<string, string>
             {
-                { "app_id", Options.App_Id },
+                { "app_id", Options.ClientId },
                 { "redirect_uri", redirectUri },
                 { "perms", scopes }
             };
 
             if (Options.UsePkce)
             {
                 byte[] bytes = new byte[32];
-                CryptoRandom.GetBytes(bytes);
+                RandomNumberGenerator.Fill(bytes);
                 string codeVerifier = Microsoft.AspNetCore.WebUtilities.Base64UrlTextEncoder.Encode(bytes);
 
                 // Store this for use during the code redemption.
                 properties.Items.Add(OAuthConstants.CodeVerifierKey, codeVerifier);
 
-                using var sha256 = SHA256.Create();
+                //using var sha256 = SHA256.Create();
+                using var sha256 = HashAlgorithm.Create("SHA256"); // Why?
```

Comment:
You can remove the comment - https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/pull/373#discussion_r362831534